content/millennium/r4/general-clinical/condition.md

```diff
@@ -73,7 +73,70 @@ _Implementation Notes_
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Update
+
+Update an existing condition
+
+    PUT /Condition/:id
+
+_Implementation Notes_
+
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json', 'If-Match': 'W/"&lt;Current version of the Condition resource>"'} %>
+
+### Body fields
+
+Notes:
+
+* `notes` cannot be added if `notes` already exists. An existing `notes` field cannot be modified.
+* `clinicalStatus` SHALL be present if `verificationStatus` is not `entered-in-error` and `category` is `problem-list-item`.
+* If condition is `abated`, then `clinicalStatus` must be either `inactive`, `resolved`, or `remission`.
+* `clinicalStatus` SHALL NOT be present if `verificationStatus` is `entered-in-error`.
+
+<%= definition_table(:condition, :update, :r4) %>
+
+### Example
+
+#### Request
+
+    PUT /Condition/p10070301
```

Comment:
this should be a fully qualified request.

Reply:
Updated [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-c91ea4243fc73689536dc746c972a3d6R109).